### PR TITLE
FH-3604 - Add command `fhc connections` to V3 standard and add unit test

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/FH-3604

Following the changes made.

- Add it to the V3 standard
- Add unit test

Following local tests

To check the help. 
`````
Camilas-MacBook-Pro:fh-fhc cmacedo$ ./bin/fhc.js help connections
(node) sys is deprecated. Use util instead.
fhc connections <action>
Where <action> is one of:
  list               	List connections
  read               	Read a connection
  update             	Update connections
Camilas-MacBook-Pro:fh-fhc cmacedo$ 

``````

`````
Camilas-MacBook-Pro:fh-fhc cmacedo$ ./bin/fhc.js help connections list
(node) sys is deprecated. Use util instead.
Usage:
 fhc connections list --project=<project>

Examples:
  fhc connections list --project=<project>    List all connections of the <project>


Options:
  --project, -p  Unique 24 character GUID of the project  [required]

Missing required arguments: project
Usage:
 fhc connections list --project=<project>

Examples:
  fhc connections list --project=<project>    List all connections of the <project>


Options:
  --project, -p  Unique 24 character GUID of the project  [required]

Camilas-MacBook-Pro:fh-fhc cmacedo$ 

``````

To check the list command 

screen shot 2017-05-25 at 12 46 54 pm

To check the read command 

screen shot 2017-05-25 at 12 48 31 pm

To check the update without env
screen shot 2017-05-25 at 12 50 35 pm

To check the update with env

screen shot 2017-05-25 at 12 50 47 pm

### CODE REVIEW

**Comment by jasonmadigan** [comment]
@camilamacedo86 

* Reviewed the code
* Ran the new unit tests via `npm test`
* Checked out locally and tested commands above against a single-node cluster, all seemed to work correctly

File: lib/cmd/fh3/connections/list.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  'desc' : i18n._('List connections'),
+  'examples' :
+  [{
+    cmd : 'fhc connections list --project=<project>',
+    desc : i18n._('List all connections of the <project>')
+  }],
+  'demand' : ['project'],
+  'alias' : {
+    'project' : 'p',
+    0 : 'project',
```
Comment (fh-qube) [inline lib/cmd/fh3/connections/list.js]: MAJOR Remove this trailing comma. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComma)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**